### PR TITLE
883478 How to change cursor and add tool-tip when mouse over the uri annotation in PdfViewer

### PR DESCRIPTION
**Pull request by KrithikaGanesan**
How to change cursor and add tool-tip when mouse over the uri annotation in PdfViewer

### CODE REVIEW

File: Annotations/Uri Annotation/UriAnnotation/MainWindow.xaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿<Window xmlns:PdfViewer="clr-namespace:Syncfusion.Windows.PdfViewer;assembly=Syncfusion.PdfViewer.WPF"  x:Class="UriAnnotation.MainWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:UriAnnotation"
+        mc:Ignorable="d"
+        Title="MainWindow" Height="450" Width="800" >
```
Comment (RajaVigneshB) [inline Annotations/Uri Annotation/UriAnnotation/MainWindow.xaml:8]: Make the window open in maximized state

File: Annotations/Uri Annotation/UriAnnotation/MainWindow.xaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="50"></RowDefinition>
+            <RowDefinition ></RowDefinition>
+        </Grid.RowDefinitions>
+        <StackPanel Orientation="Horizontal">
```
Comment (RajaVigneshB) [inline Annotations/Uri Annotation/UriAnnotation/MainWindow.xaml:14]: Align the button to the center

**Review by RajaVigneshB — CHANGES_REQUESTED** [review]

File: Annotations/Uri Annotation/UriAnnotation/MainWindow.xaml (hunk 1 above)
Comment (KrithikaGanesan) [reply Annotations/Uri Annotation/UriAnnotation/MainWindow.xaml]: I have done the changes as suggested

File: Annotations/Uri Annotation/UriAnnotation/MainWindow.xaml (hunk 2 above)
Comment (KrithikaGanesan) [reply Annotations/Uri Annotation/UriAnnotation/MainWindow.xaml]: I have done the changes as suggested

File: Annotations/Uri Annotation/UriAnnotation/MainWindow.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+            uriAnnotation.Text = "Uri Annotation";
+            //Add this annotation to a new page.
+            docView.LoadedDocument.Pages[0].Annotations.Add(uriAnnotation);
+            //Save the document to disk.
+            MemoryStream stream = new MemoryStream();
+            lDoc.Save(stream);
+            docView.Load(stream);
+            annotationPageIndex = 0;
+            annotationBounds = uriAnnotation.Bounds;
+            //annotationBounds = convertor.ConvertToPixels(annotationBounds, PdfGraphicsUnit.Point);
```
Comment (Deepak1799) [inline Annotations/Uri Annotation/UriAnnotation/MainWindow.xaml.cs:61]: * Remove the commented lines
* Remove unwanted namespaces
* Add comments wherever possible as it is a public repo

**Review by Deepak1799 — CHANGES_REQUESTED** [review]
Requested changes

File: Annotations/Uri Annotation/UriAnnotation/MainWindow.xaml.cs (hunk 3 above)
Comment (KrithikaGanesan) [reply Annotations/Uri Annotation/UriAnnotation/MainWindow.xaml.cs]: I have addressed the provided feedbacks.

File: Annotations/Uri Annotation/UriAnnotation/packages.config [hunk 4]
```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Syncfusion.Compression.Base" version="26.2.5" targetFramework="net48" />
```
Comment (Deepak1799) [inline Annotations/Uri Annotation/UriAnnotation/packages.config:3]: Why we specifically mention the version in the package? Can this be "*"?

**Review by Deepak1799 — CHANGES_REQUESTED** [review]
Requested changes

File: Annotations/Uri Annotation/UriAnnotation/packages.config (hunk 4 above)
Comment (KrithikaGanesan) [reply Annotations/Uri Annotation/UriAnnotation/packages.config]: I have addressed the provided feedback

**Review by Deepak1799 — APPROVED** [review]
Approved

**Pull request merged**